test/testcafe/spec/ChallengeCustomAppPush_spec.js

```diff
@@ -67,6 +77,41 @@ test
     await t.expect(challengeCustomAppPushPageObject.getSignoutLinkText()).eql('Back to sign in');
   });
 
+test
+  .requestHooks(pushAutoChallengeMock)('challenge custom app push screen has right labels, custom logo and auto challenge checkbox', async t => {
+    const challengeCustomAppPushPageObject = await setup(t);
+    await checkConsoleMessages({
+      controller: 'mfa-verify',
+      formName: 'challenge-poll',
+      authenticatorKey: 'custom_app',
+      methodType: 'push',
+    });
+
+    const pageTitle = challengeCustomAppPushPageObject.getPageTitle();
+    const pushBtn = challengeCustomAppPushPageObject.getPushButton();
+    const a11ySpan = challengeCustomAppPushPageObject.getA11ySpan();
+    const checkboxLabel = challengeCustomAppPushPageObject.getAutoChallengeCheckboxLabel();
+    const logoClass = challengeCustomAppPushPageObject.getBeaconClass();
+    await t.expect(pushBtn.textContent).contains('Push notification sent');
+    await t.expect(a11ySpan.textContent).contains('Push notification sent');
+    await t.expect(pushBtn.hasClass('link-button-disabled')).ok();
+    await t.expect(logoClass).contains('custom-app-logo');
```

Comment:
can we also check this is not true when `logoUri` is not present?

Reply:
if `logoUri` is not present, it will be considered as non-custom app . we do have a check for okta-verify [here](https://github.com/okta/okta-signin-widget/blob/0b46d2d18fc60d7ca8f811876f77d0d35f765f2f/test/testcafe/spec/ChallengeOktaVerifyPush_spec.js#L65) Isn't that sufficient ?